sympy/matrices/determinant.py

```diff
@@ -729,7 +729,7 @@ def _det_bareiss(M, iszerofunc=_is_zero_after_expand_mul):
         Defaults to ``lambda x: x.is_zero``.
 
     TODO: Implement algorithm for sparse matrices (SFF),
-    http://www.eecis.udel.edu/~saunders/papers/sffge/it5.ps.
+    https://www.eecis.udel.edu/~saunders/papers/sffge/it5.ps.
```

Comment:
The link is broken." https" does not solve the problem.

Reply:
What do you mean by broken? When I click on it i get a download, should I get the website instead?